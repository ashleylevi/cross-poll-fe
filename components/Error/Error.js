import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import { Icon } from 'react-native-elements';


const Error = () => {
  return (
    <View style={styles.container}>
      <Icon
        name='paw'
        type='font-awesome'
        color='black'
        size={36}
        iconStyle={styles.pawprint}
      />
      <Text style={styles.errorMessage}>
        Oops, your connection seems off...
        </Text>
    </View>
  )
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center'
  },
  errorMessage: {
    fontSize: 28,
    fontFamily: 'Kohinoor Bangla',
    color: 'black',
    textAlign: 'center',
  }
})

export default Error;