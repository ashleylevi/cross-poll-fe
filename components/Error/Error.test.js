import React from 'react'; 
import { shallow, mount } from 'enzyme';
import Error from './Error';

describe('error page', () => {

  it('matches snapshot', () => {
    let wrapper = shallow(<Error />)
    expect(wrapper).toMatchSnapshot(); 
  })
})