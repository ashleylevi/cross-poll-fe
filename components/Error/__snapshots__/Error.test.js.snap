// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error page matches snapshot 1`] = `ShallowWrapper {}`;
